build.sbt

```diff
@@ -191,7 +194,16 @@ def elasticsearch : String = {
 }
 
 def commonSettings: Seq[Setting[_]] = eclipseSettings ++ Seq(
-  javacOptions ++= Seq("-encoding", "UTF-8", "-source", "1.8", "-target", "1.8", "-Xlint:unchecked", "-Xlint:deprecation", "-parameters")
+  javacOptions in Compile ++= Seq("-encoding", "UTF-8", "-source", "1.8"),
+  javacOptions in (Compile, compile) ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-parameters")
+)
+
+// Include this into impl projects that use the message broker API
+// It overrides the production configuration to use a hardcoded Kafka broker
+// host and port rather than looking it up from the service locator.
+// See docs/running-in-conductr.md for details.
+def kafkaSettings: Seq[Setting[_]] = Seq(
+  BundleKeys.startCommand += "-Dlagom.broker.kafka.service-name=''"
```

Comment:
How about making the broker explicit here with a note:

```
// You may have to edit this list if your Kafka Docker container is 
// not listening on 127.0.0.1:9092
```

I'd rather add this info here than in the MD file with instructions you introduced in this PR to keep that file at a minimum. Also, when the kafka bundle for ConductR is ready it'll be possible to come back here and remove this whole setup without changing other files.

WDYT?